dj/construction/utils.py

```diff
@@ -8,61 +8,45 @@
 from sqlalchemy.orm.exc import NoResultFound
 from sqlmodel import Session, select
 
-from dj.construction.exceptions import CompoundBuildException
-from dj.errors import DJError, ErrorCode
+from dj.errors import DJError, DJErrorException, ErrorCode
 from dj.models.node import Node, NodeRevision, NodeType
 
 if TYPE_CHECKING:
-    from dj.sql.parsing.ast import Namespace
+    from dj.sql.parsing.ast import Name
 
 
-def make_name(namespace: Optional["Namespace"], name="") -> str:
-    """utility taking a namespace and name to make a possible name of a DJ Node"""
-    ret = ""
-    if namespace:
-        ret += ".".join(n.name for n in namespace.names)
-    if name:
-        ret += ("." if ret else "") + name
+def make_name(name: "Name") -> str:
```

Comment:
This can be replaced anywhere it is used I think by using combinations of `.name` and `.identifier` possible chained depending on the context I believe